usr/share/rear/verify/BORG/default/400_check_archive_access.sh

```diff
@@ -42,7 +44,11 @@ while true ; do
         (${choices[3]})
             # Abort recovery:
             abort_recreate
-            is_true "$wilful_input" && Error "User chose to abort '$rear_workflow' in ${BASH_SOURCE[0]}" || Error "Aborting '$rear_workflow' by default"
```

Comment:
In general code like
```
COMMAND1 && COMMAND2 && ... || ERROR_COMMAND
```
is not meant as if then else but as bash does work which is
that ERROR_COMMAND is run if any of the COMMANDs failed
e.g.
```
# echo foo && cat qqq && echo bar || echo error
foo
cat: qqq: No such file or directory
error
```

Some time ago I tried shellcheck but for me it results
too many false positives to be actually useful in practice for me.

E.g. I wonder what shellcheck reports for
```
test $WORD || echo "'$WORD' not a single non empty word"
```
I guess shellcheck wants to have `test "$WORD"`
but that is not what that code is meant to do.

I.e. shellcheck false positives can introduce bugs
so one has to carefully check each and everything
what shellcheck reports which is too much in practice
to be doable with reasonable effort (i.e. within reasonable time).
Of cource things would change if someone pays me for each and
every minute that I need to make code shellcheck compliant ;-)

According to my experience bash coding is tricky/hairy
and I know no tool that actually helps in practice
so one must rely on one's own mind only.

Reply:
I reverted this change and added a shellcheck disable comment. I hope you are fine with that? (What I was already at some other places, where shellcheck reported some false positives).

For here see d80cebcb4d11245be92493d012017d65d516f432.

Reply:
@flyinggreenfrog
thank you - perfectly fine with me the shellcheck disable comment!

Perhaps with such shellcheck disable comments
I may even think about shellcheck usefulness a second time
because this way it would be documented where
strange looking code is meant as is so others won't later "fix"
such code into something that does no longer work as intended.

Reply:
Yes, while implementing all the other checks of shellcheck, that were popping up, I tried to see, if it was ok to change the code, or to just add a shellcheck disable comment. Only in this case (I hope so), I was going to "far" in changing the actual code.